installer/templates/new/web/views/error_helpers.ex

```diff
@@ -26,10 +26,9 @@ defmodule <%= application_module %>.ErrorHelpers do
     #
     #     dngettext "errors", "1 file", "%{count} files", count
     #
-    Gettext.dngettext(<%= application_module %>.Gettext, "errors", msg, msg, opts[:count], opts)
-  end
-
-  def translate_error(msg) do
-    Gettext.dgettext(<%= application_module %>.Gettext, "errors", msg)
+    case Keyword.has_key?(opts, :count) do
```

Comment:
Yeah, that's why I was thinking of adding a second example:

```
  @doc """
  Translates an error message using gettext.
  """
  def translate_error({msg, opts}) do
    # Because error messages were defined within Ecto, we must
    # call the Gettext module passing our Gettext backend. We
    # also use the "errors" domain as translations are placed
    # in the errors.po file.
    if count = opts[:count] do
      # Ecto will pass the :count keyword if the error message is
      # meant to be pluralised. On your own code and templates,
      # this could be written simply as:
      #
      #     dngettext "errors", "1 file", "%{count} files", count
      #
      Gettext.dngettext(Decimals.Gettext, "errors", msg, msg, count, opts)
    else
      # If pluralization is not needed, on your own code and 
      # templates, this could be written simply as:
      #
      #     dgettext "errors", "is invalid"
      #
      Gettext.dgettext(Decimals.Gettext, "errors", msg, opts)
    end
  end
```